Logger/FileLogger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    class FileLogger : BaseLogger
```

Comment:
Specify access modifiers. The default access modifier for a class is `internal` meaning it will not be directly accessible outside of this project (and not directly available in the unit tests).

```suggestion
    public class FileLogger : BaseLogger
```